src/router.js

```diff
@@ -4,6 +4,7 @@ import Router from 'vue-router';
 import Home from './views/Home.vue';
 import Tourmap from './views/Tourmap.vue';
 import Selection from './views/Selection/index.vue';
+// import SelectionOverview from './views/SelectionOverview/index.vue';
```

Comment:
??

Reply:
This is for the eventual selectionoverview which we will be skipping for now